kuma/users/views.py

```diff
@@ -361,8 +361,7 @@ def user_detail(request, username):
     detail_user = get_object_or_404(User, username=username)
 
     if (detail_user.active_ban and not request.user.has_perm('users.add_userban')):
-        return render(request, '403.html',
-                      {'reason': 'bannedprofile'}, status=403)
+        return render(request, '404.html', status=404)
```

Comment:
Info needs to be passed to the 404 page so that it can distinguish between a banned user, a user that never existed, and the current default (navigated to missing page).  The 403 page does this by passing a ``reason``.